module/documents/actor.mjs

```diff
@@ -236,6 +237,13 @@ export default class ActorEd extends Actor {
     this.#processRoll( roll );
   }
 
+  /**
+   * 
+   * @param {Object}    itemId        Id of the item to rotate the status of
```

Comment:
Should probably have a description of the sequence that is rotated (I didn't document it, oops....)